docs/content/config/debugging.md

```diff
@@ -10,7 +10,10 @@ This page contains valuable information when it comes to resolving issues you en
 
     Please consider contributing solutions to the [FAQ][docs-faq] :heart:
 
-## Preliminary Information
+## Preliminary Checks
+
+- Check that all published DMS ports are actually open and not blocked by your ISP / hosting provider.
+- SSL errors are likely the result of a wrong setup on the user side and not caused by DMS itself.
```

Comment:
If either of these do present themselves as issues, either ignored here or legitimately required more info to troubleshoot, they could be expanded like the below section on outbound port 25 :+1: